code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -228,10 +228,10 @@
 	machine_current_charge = 50000 //integrated battery for recharging energy weapons. Normally 10000.
 	machine_max_charge = 50000
 
-	product_ads = "Lasgun running low? Recharge here!;Need a charge?;Power up!;Electrifying!;Empower yourself!"
+	product_ads = "Lasrifle running low? Recharge here!;Need a charge?;Power up!;Electrifying!;Empower yourself!"
 	products = list(
-					/obj/item/cell/lasgun/M43 = 10,
-					/obj/item/cell/lasgun/M43/highcap = 2,
+					/obj/item/cell/lasgun/lasrifle = 10,
+					/obj/item/cell/lasgun/lasrifle/highcap = 2,
```

Comment:
sort out the indent

Reply:
What about it?

This is the exact same indent as it's always been.

Reply:
No point having it so far indented, it just makes reviews harder.

Reply:
It would make the code less pretty though

![image](https://user-images.githubusercontent.com/45076386/79395385-204ff680-7f3f-11ea-821e-48b48886c9e8.png)

Reply:
Secondly, how are you doing your reviewing?

![image](https://user-images.githubusercontent.com/45076386/79395889-42964400-7f40-11ea-9ece-fc7fc3d1c472.png)

This seems easy enough to follow from my perspective.

Reply:
I'm pretty sure everything in that file is indented that way, would need to change everything else.

Reply:
Everything in that file is indented like this. It'd look out of place if it's just this box. I'd argue.

Reply:
Everything in all files are indented like this.

Reply:
Should be a seperate PR then, if he wants it changed.

Reply:
This is what im trying to avoid 
![image](https://user-images.githubusercontent.com/1134201/79475585-c2371800-7fff-11ea-99a1-eb1f4bb85317.png)

I wont block merging and would just end up doing this myself later.